src/main/ruby/post-boot/post-boot.rb

```diff
@@ -6,6 +6,9 @@
 # GNU General Public License version 2
 # GNU Lesser General Public License version 2.1
 
+delay do
+# rubocop:disable Style/IndentationWidth
```

Comment:
This is to avoid the RuboCop warning about bad indentation.
Fixing it would require re-indenting the entire file, even though we should have finer-grained `delay` in this file soon.

Reply:
I'd prefer re-indenting.